Source/WebCore/page/Quirks.h

```diff
@@ -211,6 +212,7 @@ class Quirks {
     mutable std::optional<bool> m_allowLayeredFullscreenVideos;
 #endif
     mutable std::optional<bool> m_hasBrokenPermissionsAPISupportQuirk;
+    mutable std::optional<bool> m_shouldEnableApplicationCacheQuirk;
```

Comment:
Can be guarded by `#if PLATFORM(IOS_FAMILY)` I believe.

Reply:
Updated; thanks for the review